jenkins_integration/Jenkinsfile

```diff
@@ -98,6 +98,18 @@ pipeline
                 }
             }
         }
+        stage('Code Size Analysis') {
+            when {
+                expression { env.BRANCH_NAME == "main" || env.BRANCH_NAME.startsWith("release_") }
```

Comment:
I don't know if we want to do it for main. We can check with Marius what he thinks.